From 33ea2adf01e9e2ce9f1e33779f0b1ac0d9d1989c Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Mon, 24 Jan 2022 09:03:29 -0500
Subject: Trac #33226: update doctests for giac-1.7.0.45.

We have two failing doctests with giac-1.7.0.45: one is a trivial
tolerance issue, but the other is a symbolic integration result for
which giac is no longer willing to return a symbolic answer. We add
"abs tol" to the numeric test, and drop the symbolic one entirely.
---
 src/sage/functions/min_max.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/sage/functions/min_max.py b/src/sage/functions/min_max.py
index fc18373..9b7d6d9 100644
--- a/src/sage/functions/min_max.py
+++ b/src/sage/functions/min_max.py
@@ -228,15 +228,15 @@ class MaxSymbolic(MinMax_base):
             ...
             TypeError: cannot evaluate symbolic expression numerically
 
-        ::
+        We can usually integrate these expressions, but can't
+        guarantee a symbolic answer in closed form::
 
             sage: f = max_symbolic(sin(x), cos(x))
             sage: r = integral(f, x, 0, 1)
             ...
-            sage: r
-            sqrt(2) - cos(1)
-            sage: r.n()
+            sage: r.n()  # abs tol 1e-8
             0.873911256504955
+
         """
         return max_symbolic(args)
 
-- 
cgit v1.0-1-gd88e

